content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -118,19 +205,49 @@ Content-Type: application/json
 }
 ```
 
-`Bookmark` events can be requested by `allowWatchBookmarks=true` option in watch requests, but clients shouldn't assume bookmarks are returned at any specific interval, nor may they assume the server will send any `bookmark` event.
+As a client, you can request `BOOKMARK` events by setting the
+`allowWatchBookmarks=true` query parameter to a **watch** request, but you shouldn't
+assume bookmarks are returned at any specific interval, nor can clients assume that
+the API server will send any `BOOKMARK` event even when requested.
 
 ## Retrieving large results sets in chunks
```

Comment:
nit: Future PR, possibly change this heading.
Chunk large result sets
Pagination